### PR TITLE
schema: broadcast changes

### PR DESCRIPTION
**Pull request by oliver-sanders**
* Fix the default value for `mode`.
* Change `settings` to use a new special input type (so the UI can display it differently).

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.
- [x] No dependency changes.

### CODE REVIEW

File: cylc/flow/network/schema.py [hunk 1]
```diff
@@ -1219,6 +1219,23 @@ def description(self):
...
 
+class BroadcastSetting(GenericScalar):
+    """A [runtime] key=value configuration for a namespace.
+
+    Should be a key=value pair where sections are wrapped with square
+    brackets.
+
+    Examples:
+        script=true
+        [environment]ANSWER=42
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1230]: These examples are true of the user input (both in the UI and on the CLI), however, the transfer format and GraphiQL differ which makes this rather awkward to document.

**Review by kinow — COMMENTED** [review]
I think two functional tests jobs failed. Looked at one of them to confirm it was due to test failures, and not other CI issue.

```
-------------------
tests/f/reload/04-inheritance.t          (Wstat: 0 Tests: 2 Failed: 1)
  Failed test:  2
tests/f/reload/08-cycle.t                (Wstat: 0 Tests: 2 Failed: 1)
  Failed test:  2
tests/f/reload/12-remove-task.t          (Wstat: 0 Tests: 2 Failed: 1)
  Failed test:  2
tests/f/reload/16-remove-add-alter-task.t (Wstat: 0 Tests: 2 Failed: 1)
  Failed test:  2
Files=4, Tests=8, 191 wallclock secs ( 0.03 usr  0.00 sys + 13.07 cusr  1.41 csys = 14.51 CPU)
    suite logs can be found under: /home/runner/cylc-run/cylctb-20201209T193759Z/f/reload/16-remove-add-alter-task/log/suite/
Result: FAIL
Error: Process completed with exit code 1.
```

The rest looks OK in this PR :+1:

**Comment by oliver-sanders** [comment]
Should be fixed.

File: cylc/flow/network/schema.py [hunk 2]
```diff
@@ -1690,30 +1709,43 @@ class Arguments(TaskMutation.Arguments):
...
+
+    """
+    return cls.Field(description=cls._meta.description)
+
+
 # Mutation declarations
 
 class Mutations(ObjectType):
     # workflow actions
-    broadcast = Broadcast.Field(description=Message._meta.description)
```
Comment (oliver-sanders) [inline cylc/flow/network/schema.py:1697]: The reason I added the `_mut_field` thing can be found on this line where the `broadcast` mutation was accidentally given the `message` metadata.

**Review by kinow — APPROVED** [review]

**Review by hjoliver — APPROVED** [review]
@dwsutherland - do you have time for a quick sanity check of this (schema changes)?

**Comment by hjoliver** [comment]
Two approvals

**Pull request merged**